guides/common/modules/proc_configuring-server-for-use-with-tftp.adoc

```diff
@@ -1,41 +1,53 @@
-[id="configuring-external-tftp_{context}"]
-= Configuring {ProductName} with external TFTP
+[id="configuring-server-for-use-with-tftp"]
+= Configuring {ProductName} for use with tftp
 
 You can configure {ProductName} with external TFTP services.
 
+
+.Prerequisites
+* You configured the TFTP server.
+* You shared the `/var/lib/tftpboot` on the TFTP server with NFS.
```

Comment:
The fstab line uses `/exports/var/lib/tftpboot` but I think the path here should match.